src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.CodeGeneration.targets

```diff
@@ -17,12 +17,12 @@ Copyright (c) .NET Foundation. All rights reserved.
   -->
 
   <UsingTask
-    TaskName="Microsoft.AspNetCore.Razor.Tasks.RazorGenerate"
+    TaskName="Microsoft.AspNetCore.Razor.Tasks.SdkRazorGenerate"
```

Comment:
2.1 Razor.Design defines tasks with these names and they're loaded prior to the SDKs. Simply renaming disambiguates the tasks and we ignore the tasks from the package.